### ISSUE TITLE
Move Climsoft.org to GitHub pages

### ISSUE THREAD

**Issue by isedwards** [opencdms/Climsoft#502]
We need to extend the Climsoft website to have multiple pages (hosted for free at GitHub).

New pages will include the 5 year roadmap and details on when each section of the CDMS Specifications will be implemented.

**Comment by Steve-Palmer** [comment]
Sounds sensible. We need to set up the web pages, then it is easy to change from the single-page site. I have the login on UK2 to change the DNS destination. Who should do  the website design?

**Comment by mhabimana** [comment]
@isedwards :we need to agree on the way forward.

**Comment by isedwards** [comment]
Yes, we're agreed to do this. It's just waiting for me to make a copy of the existing website within the `climsoft.org` repository at GitHub.

Can you add a v4.2 label?
